tests/manage/nfs_feature/test_nfs_feature_enable_for_ODF_clusters.py

```diff
@@ -0,0 +1,449 @@
+import pytest
+import logging
+import yaml
+
+
+from ocs_ci.ocs import api_client, constants, ocp
+from ocs_ci.helpers import helpers
+from functools import partial
+from ocs_ci.framework.testlib import (
+    skipif_ocs_version,
+    ManageTest,
+    tier1,
+    skipif_ocp_version,
+    skipif_managed_service,
+    skipif_disconnected_cluster,
+    skipif_proxy_cluster,
+)
+
+from ocs_ci.ocs.resources import pod
+from ocs_ci.helpers.helpers import (
+    create_pods,
+)
+
+log = logging.getLogger(__name__)
+
+
+@tier1
+@skipif_ocs_version("<4.11")
+@skipif_ocp_version("<4.11")
```

Comment:
ODF-4.11 should also function on OCP-4.10, I think?

Reply:
As per my understanding nfs feature is available from ODF 4.11. For ODF 4.10 the patch requests to enable nfs feature and rok_csi_nfs do not make any changes.

Reply:
It seems right to skip for anything below 4.11 to me.